caproto/server/common.py

```diff
@@ -704,29 +704,21 @@ def __getitem__(self, pvname):
         while True:
             # This queue receives updates that match the db_entry, data_type
             # and mask ("subscription spec") of one or more subscriptions.
-            sub_specs, metadata, values, flags, sub_id = await self.subscription_queue.get()
+            sub_specs, metadata, values, flags, sub = await self.subscription_queue.get()
 
             subs = []
-            if sub_id is None:
+            if sub is None:
                 # Broadcast to all Subscriptions for the relevant
                 # SubscriptionSpec(s).
                 for sub_spec in sub_specs:
                     subs.extend(self.subscriptions[sub_spec])
             else:
-                # A specific sub_id has been specified, which means this update
-                # is due to a *new* Subscription, and should only go to that
-                # specific Subscription. Other Subscriptions have already been
-                # sent this update and should not receive it again. Find which
-                # Subscription that is....
+                # A specific Subscription has been specified, which means this
+                # specific update was prompted by Subscription being new, not
+                # prompted by a new value. The update should only be sent to
+                # that specific Subscription.
+                subs = (sub,)
```

Comment:
super nit-pick, make this a list to match the type used above?  Or replace the above with a tuple comprehension?

Reply:
Reasonable.